README.md

```diff
@@ -53,3 +53,107 @@ python run_bot.py
 - ssh-keyscan -H <IP address/Hostname> >> ~/.shh/known_hosts
 - sudo -H pip install -U pipenv
+
+## Techstack
+# Programming Language: 
+  -Python 3.6+
+# Libraries:
+  -pipenv: For virtual environment and dependency management
+  -python-telegram-bot: For interacting with Telegram APIs
+  -Reverse Image Search Engines:
+    -Google
+    -Bing
+    -Yandex
+    -TinEye
+    -IQDB
+# Platform:
+  -MS Azure for hosting
+# Databases:
+  -SQLite (for user-specific data like search history, preferences)
+
+### Repository Structure 📂
+
+```plaintext
+ReVot/
+│
+├── .github/                  # GitHub-specific files
+│   └── ISSUE_TEMPLATE/
+│       ├── bug_report.md     # Template for reporting bugs
+│       ├── custom.md         # Custom issue template
+│       └── feature_request.md # Template for requesting features
+│
+├── deploy/                   # Deployment scripts and configurations
+│   ├── after_push            # Post-deployment scripts
+│
+├── reverse_image_search_bot/ # Main bot directory
+│   ├── uploaders/            # Uploader modules
+│   │   ├── __init__.py       # Initialize uploaders package
+│   │   ├── base_uploader.py   # Base class for uploaders
+│   │   ├── file_system.py     # File system operations
+│   │   └── ssh.py            # SSH related functions
+│   │
+│   ├── __init__.py           # Initialize bot package
+│   ├── bot.py                # Main bot logic
+│   ├── commands.py           # Command handling for the bot
+│   ├── image_search.py       # Functions for reverse image search
+│   ├── settings.example.py    # Example settings file for API tokens
+│   ├── settings.example1.py   # Another example settings file
+│   ├── utils.py              # Utility functions and helpers
+│
+├── LICENSE                   # License information
+├── Pipfile                   # Pipenv dependencies
+├── Pipfile.lock              # Locked dependency versions
+├── README.md                 # Main project documentation
+└── run_bot.py                # Script to run the bot
+
+```
+
+# How to Contribute 🌍
+We welcome contributions from the community! Here's how you can contribute:
+
+1. **Fork the Repository**: Click on the "Fork" button at the top of this repository.
+
+2. **Clone Your Fork**: Clone your forked repo to your local machine.
+   ```bash
+   git clone https://github.com/your-username/ReVot.git
+   cd ReVot
+   ```
+
+3. **Create a Branch**: Create a new branch for your feature or bug fix.
+   ```bash
+   git checkout -b feature-name
+   ```
+
+4. **Make Changes**: Implement your changes and commit them.
+   ```bash
+   git add .
+   git commit -m "Add a detailed description of your changes"
+   ```
+
+5. **Push to Your Fork**:
+   ```bash
+   git push origin feature-name
+   ```
+
+6. **Submit a Pull Request**: Go to the original repository and submit a PR from your fork.
+
+We will review your submission and provide feedback.
+
+# Contribution Guidelines 📝
```

Comment:
Just give an overview in this section and refer this section to `CODE_OF_CONDUCT.md`